service-api/src/main/java/greencity/dto/TariffsForLocationDto.java

```diff
@@ -12,10 +12,8 @@
 @Builder
 public class TariffsForLocationDto {
     private Long tariffInfoId;
-    private Long minAmountOfBigBags;
-    private Long maxAmountOfBigBags;
-    private Long minPriceOfOrder;
-    private Long maxPriceOfOrder;
+    private Long minQuantity;
```

Comment:
This DTO is used in `@GetMapping("/orders/{id}/tariff")` endpoint
Have you checked that `the UI` components that use this endpoint work fine after these changes?